src/insi-client.service.ts

```diff
@@ -271,4 +273,28 @@ export class INSiClient {
       this._overrideHttpClientResponse('./fixtures/TEST_2.08_cas2.xml');
     }
   }
+
+  public static getClientWithDefinedId = (idam: string, overrideSpecialCases = true, softwareVersion?: string, softwareName?:string, emitter?: string): INSiClient => {
+    const lps = new LPS({
+      idam,
+      version: softwareVersion ? softwareVersion : SOFTWARE_VERSION,
+      name: softwareName ? softwareName : SOFTWARE_NAME,
+      id: 'b3549edd-4ae9-472a-b26f-fd2fb4ef397f',
+    });
+  
+    const lpsContext = new LpsContext({
+      emitter: emitter ? emitter : 'medecin@yopmail.com',
```

Comment:
prefer using a padoa branded email like `previsit+certificate-testing-script@padoa.fr` so we get the email if they ever get in touch